.github/actions/setup-tauri/action.yml

```diff
@@ -23,8 +23,14 @@ runs:
       if: ${{ runner.os == 'Linux' }}
       run: sudo apt-get install at-spi2-core xvfb
       shell: bash
-    - name: Download WebView2 bootstrapper
+    - uses: actions/cache@v4
       if: ${{ runner.os == 'Windows' }}
+      id: cache-webview2-installer
+      with:
+        path: WebView2Installer.exe
+        key: ${{ runner.os }}-webview2-installer
```

Comment:
The arch should be here too, it'll save a little trouble whenever aarch64 Windows comes up.

Most other cache keys, like for Rust, are OS + arch, right?